### ISSUE TITLE
Docker container

### ISSUE THREAD

**Issue by ewoks** [djotaku/Makemkv-Spec-for-Fedora#10]
Any blockers to do this in a docker container so the host system skip any additional installation?

**Comment by djotaku** [comment]
Nope. Do you have instructions for that? We can add to the README

**Comment by ewoks** [comment]
basically it is about adding minimal Containerfile (Dockerfile) to the repo, so everyone can build it and scripts will be executed inside container

**Comment by djotaku** [comment]
Yeah, I'd be OK with adding a Dockerfile to the repo. I'm not interested in creating it myself at this time. But if you (or someone) made a Pull Request, I'd add it (after review, of course)
